### PR TITLE
Tangled blog app: Set up views

### PR DESCRIPTION
**Pull request by angeluray**
## Hi good reviewer, it's nice to see you again 🖖🏼 🤖 

Happy Greeting

## Basic update check ✔️ 

- Created a new branch according to GitHub flow rules.
- Add views for each route.

**Note:**  *If you have further questions about the project or you see something that could be fixed without asking for a new review just let me know or mark it as optional changes, I'll 100% consider your suggestions and I'll be completely willing to fix it 😃 .*

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,42 @@
...
+
+          <div>
+            <img class='post-counter-icon' src="https://img.icons8.com/sf-ultralight/50/null/good-quality.png"/>
+            <p class='post-counter'><%= post.likes_counter %></p>
+          </div>
+        </div>
+
+        <div class='comments-section'>
+        <% post.comments.each do |comment| %>
+          <p><%= comment.author.name %>: <%= comment.text %></p>
```
Comment (omar-muhamad) [inline app/views/posts/index.html.erb]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use the `check_last_five_comments` method that you created in the post model here to render the 5 most recent comments only.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @angeluray,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] You have a problem with your migration files, You remove references that you didn't create yet and you drop tables that don't exist. So please fix that because now when I try to run your app migrations files I have a lot of errors.
[Hint] You can fix that by just removing the migration files I marked in the image below

image

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (angeluray) [reply app/views/posts/index.html.erb]: Ok, thanks @Omar-Muhamad . This is how that part of the project looked like before making those changes:

image

This is how it looks like after making the changes:

image

Now it displays the last 5 comments.

**Review by m05e5 — APPROVED** [review]
Hi @angeluray , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
goodjob

## Highlights
✔️ your linters are all ok
✔️ Descriptive PR 

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**